lib/ec2imgutils/ec2deprecateimg.py

```diff
@@ -249,6 +240,7 @@ def deprecate_images(self):
             'Replacement image {}'.format(self.replacement_image_tag)
         )
 
+        ec2 = self._connect()
```

Comment:
Unrelated to the PR but it'd be nice for the attr to be lazy and obfuscate the need to "connect" inside an attr method.